datalad/interface/tests/test_run.py

```diff
@@ -920,6 +920,49 @@ def test_run_explicit(path):
     ok_(ds.repo.file_has_content(opj("subdir", "foo")))
 
 
+@known_failure_direct_mode
```

Comment:
should it also get a `@known_failure_windows` due to use of `echo o >> foo`?  related fresh #3501

Reply:
I wrote that in a way that I hope passes on Windows, although even so the large rerun will probably fail on Windows.  I don't plan on marking it as a known Windows failure until I actually know it fails.  That won't be clear until this is merged to master.  If a fix is needed on top, I'll open the merge + fix as a PR.

Reply:
well, we wouldn't know I am afraid since here appveyor fails early on `datalad.distribution.tests.test_install.test_datasets_datalad_org` and never runs `test_run` tests -- the curse of letting CI failures (in particular appveyor) to get "stale", heh heh

Reply:
> well, we wouldn't know I am afraid since here appveyor fails early on `datalad.distribution.tests.test_install.test_datasets_datalad_org` and never runs `test_run` tests

That seems odd given their positions in master's appveyor.yml (test_install is last), but ok.   If needed, I can push a scratch branch with the tests pruned down, so it is still easy enough to check.

> -- the curse of letting CI failures (in particular appveyor) to get "stale", heh heh

Sadly I don't think anyone was able to understand the transient test_datasets_datalad_org failures.  But perhaps we should mark it as a known failure if it's blocking other tests on appveyor.

Reply:
> I can push a scratch branch with the tests pruned down, so it is still easy enough to check.

I've did that here: <https://ci.appveyor.com/project/mih/datalad/build/job/oa0won9v6n78qrl0>.  Initial command runs fine, but it hits the same error as direct mode with the "master~" ref.

I've updated this PR to mark it as a `@known_failure_windows`.